docs/getting-started/server/sso/local.md

```diff
@@ -13,31 +13,31 @@ This uses
     - API
     - SSO (located at `server/bitwarden_license/src/Sso`)
 
-2.  Local web client running
+2.  Local web client running.
 
 ## Configure IdP
 
-1.  Open your local web vault and navigate to your organization → Settings → Single Sign-On
+1.  Open your local web client and navigate to your organization → Settings → Single Sign-On.
 
-2.  Tick the "Allow SSO authentication" box
+2.  Tick the "Allow SSO authentication" box.
 
-3.  Come up with and enter an SSO Identifier
+3.  Come up with and enter an SSO Identifier.
 
 4.  Select "SAML 2.0" as the SSO type. Don't save or exit this page yet, you'll need to come back to
-    it later
+    it later.
 
 5.  Open a new terminal and navigate to the `dev` folder in your server repository, e.g.
 
     ```bash
     cd ~/Projects/server/dev
     ```
 
-6.  Open your `.env` file and set the following environment variables, based on the "SP Entity ID"
-    and "Assertion Consumer Service (ACS) URL" values on the web vault SSO configuration page:
+6.  Open your `.env` file and set the following environment variables using the "SP Entity ID" and
+    "Assertion Consumer Service (ACS) URL" values from the SSO configuration page opened in step #2:
```

Comment:
Question:  What step are you referencing in: `opened in step #2` ?  Step 2 is `Tick the "Allow SSO authentication" box.` within this section?

Reply:
Oops, I put #2 instead of #4.  I'e fixed it.